src/com/dabomstew/pkrandom/RomFunctions.java

```diff
@@ -39,22 +39,22 @@
 
 public class RomFunctions {
 
-    public static Set<Pokemon> getBasicOrNoCopyPokemon(RomHandler baseRom) {
+    public static Set<Pokemon> getBasicPokemon(RomHandler baseRom, boolean includeNonCopyingMons) {
         List<Pokemon> allPokes = baseRom.getPokemonInclFormes();
-        Set<Pokemon> dontCopyPokes = new TreeSet<>();
+        Set<Pokemon> basicPokemon = new TreeSet<>();
```

Comment:
I'm not sure about the method/variable renaming here. This method is only used in `copyUpEvolutionsHelper`, where its return value is set to a variable named `dontCopyPokes`, and the point of the method is explicitly to find the Pokemon for which something should not be copied (which includes basic Pokemon, and in other cases also split evolutions). In other words, the focus of the method is to find "No Copy Pokemon", not "Basic Pokemon". I'd like to have the method name and the name of this Set changed back, and the new input parameter could just be named `dontCopySplitEvos`.

Reply:
Changed it back. I mostly just renamed it since I thought I could shorten it by having the copying part in the parameters, but based on the usage it does make a lot of sense the way it was before.

Also, looking at this again, I realize that `getMiddleEvolutions` and `getFinalEvolutions` also account for split evolutions in the same way. I'll add the parameter to these methods too, mostly so I can fix the `toMonIsFinalEvo` boolean in case it needs to be used in the future with `dontCopySplitEvos` enabled.